gradle.properties

```diff
@@ -10,7 +10,8 @@ mlTestRestPort=8042
 # paths to deploy for development
 mlModulePaths=src/main/ml-modules,src/test/ml-modules
 
-pkg_version=0.0.14
+group=com.marklogic.community
+version=0.1.DEV
```

Comment:
why "0.1.DEV"? while we haven't been real good about semantic versioning yet, that's what we're intending.

Reply:
I didn't want to use a real version number here as I wasn't sure what to propose. I'm open for anything, I just didn't want to lay claim to e.g. "0.1.0".

Reply:
Let's go with 0.0.15 for now. I figure we'll do a release at the end of this sprint (next week) and will probably bump it to 0.1.0 then, but for now, let's keep it in line with the trend.